cpg_workflows/workflow.py

```diff
@@ -697,13 +698,14 @@ def _is_reusable(self, expected_out: ExpectedResultT) -> tuple[bool, Path | None
                 Path | None: first missing path, if any
         """
         if self.assume_outputs_exist:
-            logging.info(f'Assuming outputs exist. Expected output is {expected_out}')
+            # change this to debug, as it's a bit noisy
```

Comment:
This is a comment for a commit message, not a source code comment — It doesn't make sense without the context of the change.